### ISSUE TITLE
[ Feature Request ] Compilation Performance

### ISSUE THREAD

**Issue by Jinkxed** [cstgrid/k8comp#7]
First off let me say that I absolutely love this plugin.  It's central to how we are managing our environments with helm and kubernetes.

We are heavily utilizing this plugin for all of our helm deployments.

One thing we are noticing is the time it takes to create the templates.  We have some templates with quite a few variables and this causes k8comp to slow down dramatically.  

Most runs takes around 3-4 minutes to compile both in our CircleCI environment and even locally on a 64gb Macbook Pro.

I'm curious if you are aware of any possible improvements to the script to improve compilation times?

Another alternative I have been thinking about is looking to port it to Go.  Would this be something you would be interested in?

**Comment by catalinpan** [comment]
I agree with you that there are some performance issues when it comes to large files with many variables. I've started a while ago the work on a Go version but I never had the time to add all the options currently available in the current script. As soon as I will be able to complete everything I will push the Go version to this repository.

**Comment by Jinkxed** [comment]
Any progress on that go version? :)
